common/config/rush/browser-approved-packages.json

```diff
@@ -9,6 +9,10 @@
     {
       "name": "react-dom",
       "allowedCategories": [ "tests" ]
+    },
+    {
+      "name": "sass",
+      "allowedCategories": [ "libraries" ]
     }
```

Comment:
This should go in `nonbrowser-approved-packages.json`

Reply:
OK I'll review and make the change. FWIW this was autogenerated by tooling.

Reply:
Resolved in https://github.com/microsoft/rushstack/pull/2682/commits/7977a5eed2b23b9b24a4419ca047a402d45a5b2c